setup.sh

```diff
@@ -52,19 +52,19 @@ function check_dependencies()
   gentoo=("qlist -I " "emerge")
   fedora=("rpm -q " "dnf install -y")
   declare -A distro_commands=(['arch']=arch ['gentoo']=gentoo ['fedora']=fedora ['debian']=debian)
-  declare -n c
+  declare -n distro_cmd
   distro=$(detect_distro '/')
 
-  for c in ${distro_commands[$distro]}; do
-    pfix=${c[1]}
-    command=${c[0]}
+  for distro_cmd in ${distro_commands[$distro]}; do
+    install_cmd=${distro_cmd[1]}
+    command=${distro_cmd[0]}
```

Comment:
Ok, now I can see that you stack some of my comments from the previous review in this new commit (I review PR from the first commit to the last one). Changing the original commit is the right way to do this sort of modification. You can achieve it by using `git rebase -i SHA_COMMIT_BEFORE_YOUR_FIRST_COMMIT`, and in the interactive menu, you should use the `e`  option for each commit that you want to make a change. After ensuring each commit is self-contained, you finish the rebase and force push to your branch.

Check this page; it might help you:
https://flusp.ime.usp.br/git/feed-english/rewriting-history-101/